landoui/assets_src/js/components/Navbar.js

```diff
@@ -0,0 +1,39 @@
+$.fn.landoNavbar = function(){
```

Comment:
IMO we shouldn't initialize components from within their own files; we should instead have a `main.js` where we initialize and use stuff.